README.md

```diff
@@ -168,6 +168,60 @@ Por convenção utilizados o seguintes campos dentro do payload de erro:
 - **reason**: mais detalhes do erro
```

Comment:
Are we giving `mais detalhes` ou o `motivo` here?

Reply:
More details, can be the reason or the error source.